README.md

```diff
@@ -19,6 +19,15 @@ Well, the idea to use useEffect hook is to execute code that needs happens durin
 
 ### Learning resources:
 <!-- Add learning resources here -->
+[Using the Effect Hook](https://reactjs.org/docs/hooks-effect.html)
```

Comment:
We can remove this resource since it has same link as the resource mentioned on line no.24